### DISCUSSION TITLE
请问ANTV X6的贝塞尔曲线是通过什么公式计算出来的一条唯一曲线

### DISCUSSION THREAD

**Discussion by lijiapeng0302** [antvis/X6#4437] [Q&A]
正常数学的贝塞尔曲线，会通过起始点、结束点、控制点以及t的变化确认一条唯一的贝塞尔曲线，但是我看ANTV X6中的贝塞尔曲线并没有控制点，只有曲线上的点，所以我想了解一下ANTV X6的贝塞尔曲线是通过什么公式计算出来的一条唯一曲线

**Comment by resetsix** [comment]
我也不是很懂，帮你问了 ai

<img width="645" alt="图片" src="https://github.com/user-attachments/assets/289e244f-a42f-4fb6-9c71-e87ca7d9cbd7">

- [antv/x6智能路由文档](https://x6.antv.antgroup.com/api/registry/router#manhattan)
- [antv/x6智能路由示例](https://x6.antv.antgroup.com/examples/edge/router/#manhattan)
